### PR TITLE
feat: create custom dynamic menus and re-format

### PR DESCRIPTION
**Pull request by wu-clan**
This PR is associated with https://github.com/fastapi-practices/fastapi_best_architecture/pull/162

### CODE REVIEW

**Comment by wu-clan** [comment]
please review.

**Comment by downdawn** [comment]
What formatting tool are you using?
Actually, the Arco project was initialized with Husky configuration, but I removed it for convenience when committing.

**Comment by wu-clan** [comment]
Yes, husky for git

The formatting of this PR is just a simple formatting based on webstom, including optimized import, sorting (alphabetical order, empty lines of code), no specific coding specification

**Comment by wu-clan** [comment]
The final specification is eslint

**Comment by wu-clan** [comment]
We can pick up husky again

**Comment by downdawn** [comment]
OK

File: src/hooks/user.ts [hunk 1]
```diff
@@ -10,8 +10,8 @@ export default function useUser() {
     await userStore.logout();
     const currentRoute = router.currentRoute.value;
     Message.success('登出成功');
-    router.push({
-      name: logoutTo && typeof logoutTo === 'string' ? logoutTo : 'login',
```
Comment (downdawn) [inline src/hooks/user.ts:14]: Here it seems that it cannot be reduced to True

**Review by downdawn — APPROVED** [review]

**Comment by downdawn** [comment]
LGTM.

File: src/hooks/user.ts (hunk 1 above)
Comment (wu-clan) [reply src/hooks/user.ts:14]: ts should not need to judge the type.

**Pull request merged**